.circleci/config.yml

```diff
@@ -3,34 +3,30 @@ executors:
   docker-publisher:
     environment:
       IMAGE_NAME: logdna/logdna-agent
+      ICR_IMAGE_NAME: icr.io/ext/logdna-agent
     docker:
       - image: circleci/buildpack-deps:stretch
 jobs:
-  test:
```

Comment:
Why are we removing test?

Reply:
I'm gonna add tests in separate `Pull Request` - wanted to keep all clean